backend/packages/Upgrade/src/api/services/MoocletExperimentService.ts

```diff
@@ -207,11 +207,12 @@ export class MoocletExperimentService extends ExperimentService {
       });
       throw error;
     }
+    if (!queries.some((query) => query?.metric?.key === rewardMetricKey)) {
```

Comment:
got it, when importing, this will obviously not need to be re-added. makes sense to me.